.gitignore

```diff
@@ -3,3 +3,4 @@ composer.lock
 composer.phar
 phpunit.xml
 php-cs-fixer.phar
+.idea/
```

Comment:
This should go in the global gitignore.

Reply:
Ah sure, I'll just remove it and update the PR then.